### DISCUSSION TITLE
v2.1.6

### DISCUSSION THREAD

**Discussion by joschi** [dropwizard/dropwizard#6921] [Announcements]
### Bug fixes

* Fix mustache docs link in views.rst by @Querela in https://github.com/dropwizard/dropwizard/pull/6916
* Use Mockito BOM in dropwizard-dependencies by @JKomoroski in https://github.com/dropwizard/dropwizard/pull/6866
* Set application state healthy if there are no unhealthy critical checks by @joschi in https://github.com/dropwizard/dropwizard/pull/6907

### Dependency updates

* Update github/codeql-action digest to 32dc499 by @renovate in https://github.com/dropwizard/dropwizard/pull/6738
* Update dependency org.apache.maven.plugins:maven-compiler-plugin to v3.11.0 by @renovate in https://github.com/dropwizard/dropwizard/pull/6745
* Update dependency org.apache.maven.plugins:maven-compiler-plugin to v3.11.0 by @renovate in https://github.com/dropwizard/dropwizard/pull/6744
* Update jetty.version to v9.4.51.v20230217 (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/6750
* Update scacap/action-surefire-report digest to a2d14ba by @renovate in https://github.com/dropwizard/dropwizard/pull/6749
* Update dependency org.checkerframework:checker-qual to v3.32.0 by @renovate in https://github.com/dropwizard/dropwizard/pull/6762
* Update dependency org.apache.tomcat:tomcat-jdbc to v9.0.73 by @renovate in https://github.com/dropwizard/dropwizard/pull/6765
* Properly declare plugin versions in java-simple archetype by @joschi in https://github.com/dropwizard/dropwizard/pull/6767
* Update dependency net.bytebuddy:byte-buddy to v1.14.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6774
* Update dependency com.uber.nullaway:nullaway to v0.10.10 by @renovate in https://github.com/dropwizard/dropwizard/pull/6780
* Update actions/cache action to v3.3.0 by @renovate in https://github.com/dropwizard/dropwizard/pull/6787
* Update actions/cache action to v3.3.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6799
* Update dependency maven-wrapper to v3.2.0 by @renovate in https://github.com/dropwizard/dropwizard/pull/6800
* Update github/codeql-action digest to 16964e9 by @renovate in https://github.com/dropwizard/dropwizard/pull/6798
* Update dependency net.bytebuddy:byte-buddy to v1.14.2 by @renovate in https://github.com/dropwizard/dropwizard/pull/6808
* Update scacap/action-surefire-report digest to 4845216 by @renovate in https://github.com/dropwizard/dropwizard/pull/6807
* Update dependency maven to v3.8.8 by @renovate in https://github.com/dropwizard/dropwizard/pull/6810
* Update dependency org.apache.maven.plugins:maven-surefire-plugin to v3 by @renovate in https://github.com/dropwizard/dropwizard/pull/6811
* Update dependency org.apache.maven.plugins:maven-surefire-plugin to v3 by @renovate in https://github.com/dropwizard/dropwizard/pull/6812
* Update actions/checkout digest to 24cb908 by @renovate in https://github.com/dropwizard/dropwizard/pull/6823
* Update github/codeql-action digest to 168b99b by @renovate in https://github.com/dropwizard/dropwizard/pull/6824
* Update actions/checkout action to v3.4.0 by @renovate in https://github.com/dropwizard/dropwizard/pull/6827
* Update dependency io.dropwizard.metrics:metrics-bom to v4.2.18 by @renovate in https://github.com/dropwizard/dropwizard/pull/6840
* Update dependency org.apache.maven.plugins:maven-help-plugin to v3.4.0 by @renovate in https://github.com/dropwizard/dropwizard/pull/6843
* Update dependency maven to v3.9.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6842
* Update dependency org.apache.maven.plugins:maven-release-plugin to v3 by @renovate in https://github.com/dropwizard/dropwizard/pull/6850
* Update dependency com.mattbertolini:liquibase-slf4j to v5 by @renovate in https://github.com/dropwizard/dropwizard/pull/6852
* Update github/codeql-action digest to 67a35a0 by @renovate in https://github.com/dropwizard/dropwizard/pull/6857
* Update actions/stale action to v8 by @renovate in https://github.com/dropwizard/dropwizard/pull/6862
* Update dependency joda-time:joda-time to v2.12.3 by @renovate in https://github.com/dropwizard/dropwizard/pull/6869
* Update dependency org.glassfish.jersey:jersey-bom to v2.39.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6870
* Update dependency joda-time:joda-time to v2.12.4 by @renovate in https://github.com/dropwizard/dropwizard/pull/6875
* Update actions/checkout digest to 8f4b7f8 by @renovate in https://github.com/dropwizard/dropwizard/pull/6874
* Update dependency org.apache.maven.plugins:maven-deploy-plugin to v3.1.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6876
* Update dependency org.apache.maven.plugins:maven-deploy-plugin to v3.1.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6877
* Update dependency org.apache.maven.plugins:maven-install-plugin to v3.1.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6878
* Update dependency org.apache.maven.plugins:maven-install-plugin to v3.1.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6879
* Update dependency org.apache.maven.plugins:maven-resources-plugin to v3.3.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6880
* Update dependency org.apache.maven.plugins:maven-resources-plugin to v3.3.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6881
* Update dependency org.apache.maven.shared:maven-filtering to v3.3.1 by @renovate in https://github.com/dropwizard/dropwizard/pull/6882
* Update actions/checkout action to v3.5.0 by @renovate in https://github.com/dropwizard/dropwizard/pull/6883
* Update logback.version to v1.2.12 (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/6871
* Update actions/setup-java digest to 5ffc13f by @renovate in https://github.com/dropwizard/dropwizard/pull/6910
* Update github/codeql-action digest to 04df126 by @renovate in https://github.com/dropwizard/dropwizard/pull/6911
* Update dependency net.bytebuddy:byte-buddy to v1.14.3 by @renovate in https://github.com/dropwizard/dropwizard/pull/6917

## New Contributors
* @JKomoroski made their first contribution in https://github.com/dropwizard/dropwizard/pull/6866
* @Querela made their first contribution in https://github.com/dropwizard/dropwizard/pull/6916

**Full Changelog**: https://github.com/dropwizard/dropwizard/compare/v2.1.5...v2.1.6

<hr /><em>This discussion was created from the release <a href='https://github.com/dropwizard/dropwizard/releases/tag/v2.1.6'>v2.1.6</a>.</em>

**Comment by erwinw** [comment]
Maybe I'm doing something wrong, but Maven Central is showing v3 & v4 as released too?

**Reply by joschi** [reply]
> Maven Central is showing v3 & v4 as released too

This is correct but we didn't announce them yet.

EDIT: Now we did.

- https://github.com/dropwizard/dropwizard/discussions/6930
- https://github.com/dropwizard/dropwizard/discussions/6931
